tests/integration/test_basic.py

```diff
@@ -189,11 +188,15 @@ def test_zipbomb_content_encoding(mini_sentry, relay, route, status_code):
                 route,
                 mini_sentry.get_dsn_public_key(project_id),
             ),
-            headers={"content-encoding": "gzip", "content-length": str(size)},
+            headers={
+                "content-encoding": "gzip",
+                "content-length": str(size),
+                "content-type": "application/octet-stream",
```

Comment:
This is an important fix. Before, we didn't set this content type, which had the following effect:
1. The minidump endpoint did not detect this as a standalone minidump
2. It went into multipart parsing, so the parser tried to find a boundary
3. There's no boundary, nor a multipart payload, so the endpoint fails immediately. 

This was the 400 status code we received, which had nothing to do with the zipbomp and content sizes. By adding the content type, we force the standalone minidump branch, and at least test that part.